torch/csrc/jit/codegen/cuda/kernel_cache.h

```diff
@@ -67,6 +74,19 @@ class TORCH_CUDA_CU_API FusionKernelRuntime {
     }
   }
 
+  //! Turn On/Off profiling
+  void profile(bool to_profile = true) {
+    profiling_ = to_profile;
+  }
+
+  //! Return the most recently used executor, corresponding to the
+  //!  most recent kernel launch.
+  ExecutorLog getMostRecentExecutorLog() {
```

Comment:
Should we instead have an interface to grab all the logs from execution?

Reply:
That's a great idea. The history/log/profile interface will be expanded for sure and currently I'm not sure how far back on the history we want to log and what we want to put in long term logs. Added a TODO in the comment and will expand it once I sort through possible use cases for this logging interface.